cmd/osbuild-composer/main.go

```diff
@@ -55,6 +59,18 @@ func createTLSConfig(c *connectionConfig) (*tls.Config, error) {
 		Certificates: []tls.Certificate{cert},
 		ClientAuth:   tls.RequireAndVerifyClientCert,
 		ClientCAs:    roots,
+		VerifyPeerCertificate: func(rawCerts [][]byte, verifiedChains [][]*x509.Certificate) error {
+			for _, chain := range verifiedChains {
+				for _, domainGlob := range c.DomainAllowlist {
+					commonName := chain[0].Subject.CommonName
+					if glob.MustCompile(domainGlob).Match(commonName) {
```

Comment:
Should we pre-compile this when loading the config? My main concern is that this would panic at runtime if the config file is wrong, would be better to error out early.

Reply:
This code was dropped.